tests/deal_ticket/test_trading_deal_ticket_submit_account.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from actions.utils import wait_for_toast_confirmation
+
+
+
+order_size = "order-size"
+order_price = "order-price"
+place_order = "place-order"
+deal_ticket_warning_margin = "deal-ticket-warning-margin"
+deal_ticket_deposit_dialog_button = "deal-ticket-deposit-dialog-button"
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_should_display_info_and_button_for_deposit(continuous_market, vega: VegaService, page: Page):
```

Comment:
if we keep both tests can we reuse the market?

Reply:
done